examples/Vendored XCFramework Example/BananaLib/DynamicFramework/CoconutLib.xcframework/ios-arm64_armv7/CoconutLib.framework/BCSymbolMaps/3A0FB3A9-0701-3DAE-81DD-C23709193E12.bcsymbolmap

```diff
@@ -0,0 +1,13 @@
+BCSymbolMap Version: 2.0
```

Comment:
should these files be gitignored?

Reply:
looks like they were not previously. What you see here is additions of new of these files due to new architectures since I rebuilt this framework using Xcode 12.

Reply:
yeah I think we use these to verify that the apps build with bitcode enabled but don't remember the specifics